### ISSUE TITLE
ADA compliance

### ISSUE THREAD

**Issue by ljgarnes** [elementor/elementor#15260]
I have a website where ADA Compliance is very important. Website Accessibility

An empty link was encountered. If a link contains no text, the function or purpose of the link will not be presented to the user. This can introduce confusion for keyboard and screen reader users. Switch to the Text view to locate and remove the empty tag or add an aria-label attribute with descriptive text. If the link includes an image that is missing alternate text, use the edit image option to add alt text to the image. Upgrade to the full version for additional options to automatically correct this issue. WCAG 2.1 (Level A) - 2.4.4

Can you add a feature for ALL links where we can include a Title Attribute or Link Label? Keep scrolling down for the example.

https://www.w3.org/WAI/fundamentals/accessibility-intro/

**Comment by rami-elementor** [comment]
Image that is missing an alternate text is not an Elementor issue. You need to go to your WordPress "Media Library" and add "Alternative text" to all your images.

To add link labels in the Elementor editor, you can simply click on the "link settings", there you will see an "attributes" field. Add `aria-label|Screen reader text` or `title|screen reader text` to make the link accessible.

**Comment by ljgarnes** [comment]
The link label wasn't always available. So thank you. This should work hopefully. The readers need the ability to work for the blind folks.

Lorelei Garnes, Msc.
50 S. Main Street
Waynesville NC 28786
(828)421-2807
________________________________
